modin/core/execution/unidist/common/engine_wrapper.py

```diff
@@ -90,6 +90,25 @@ def materialize(cls, obj_id):
         """
         return unidist.get(obj_id)
 
+    @classmethod
+    def put(cls, data, **kwargs):
```

Comment:
Can we remove **kwargs at all or it breaks the things?

Reply:
I would not want to do this, because for other engines this parameter is used and we have places where the code is common for all engines.

Reply:
Which places do you mean?

Reply:
Amendment, such places are planned, for example, for `read_csv` dispatcher.